### PR TITLE
fix: Typos and Links

### PR DESCRIPTION
**Pull request by adam-sroka**
This PR fixes minor grammar and style issues, along with updating [old links](https://github.com/jedisct1/dnscrypt-proxy) to dnscrypt-proxy GitHub repo to [new ones](https://github.com/DNSCrypt/dnscrypt-proxy).

Please review that nothing is broken as a result of these fixes and request changes if needed 😊 !

### CODE REVIEW

File: website/index.html [hunk 1]
```diff
@@ -443,7 +443,7 @@ <h2 class="w3-center" style="text-shadow:1px 1px 0 #444">FAQ</h2>
                         <p>https://gist.github.com/meanevo/e70ca58e361fb4d1a9d262a8f12b173a (HAProxy) https://stuff-things.net/2016/11/30/haproxy-sni/ https://pre-prod.chown.me/blog/running-dot-on-openbsd.html https://www.haproxy.com/blog/introduction-to-haproxy-acls/</p>
                     </div>
                     <div id="block-method" class="w3-container city w3-animate-opacity">
-                        <p>Do Blahdns block CNAME Cloacking? Yes, click here to read more.</p>
+                        <p>Does Blahdns block CNAME Cloacking? Yes, click here to read more.</p>
```
Comment (adam-sroka) [inline website/index.html:446]: There is a missing link here — one cannot click to read more.

Comment (ookangzheng) [reply website/index.html:446]: Link： https://www.ookangzheng.com/how-do-cname-cloacking-track-you/

Comment (adam-sroka) [reply website/index.html]: Link added!

**Comment by adam-sroka** [comment]
@ookangzheng Let me know if there are more changes to be made 😊 .

**Comment by ookangzheng** [comment]
LGTM

**Pull request merged**